src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
 // export const Person = ({ person }) => ();
+import '../../App.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const personPartenStatus = sex === 'm' ? 'wife' : 'husband';
+  // const personPartner = person.isMarried
+  //   ? `${person.personName} is my ${personPartenStatus}`
+  //   : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {person.age && <p className="Person__age">{`I am ${age}`}</p>}
```

Comment:
You can use only age. (not person.age).
You did the destructuring.

Reply:
Good point, fix this